bots/checkout/src/main/java/org/openjdk/skara/bots/checkout/CheckoutBot.java

```diff
@@ -64,12 +64,7 @@ private static String urlEncode(URI uri) {
     }
 
     private URI webURI() {
-        var webURI = from.webUrl().toString();
-        if (!webURI.endsWith(".git")) {
-            webURI += ".git";
-        }
-
-        return URI.create(webURI);
+        return from.remoteUrl();
```

Comment:
I meant that this whole method is redundant now and can just be inlined.

Reply:
Sorry, fixed it now.